bodhi/server/buildsys.py

```diff
@@ -179,15 +193,15 @@ def listBuildRPMs(self, id, *args, **kw):
                  'epoch': None,
                  'id': 62331,
                  'name': 'TurboGears',
-                 'nvr': 'TurboGears-1.0.2.2-2.fc7',
+                 'nvr': 'TurboGears-1.0.2.2-2.fc17',
                  'payloadhash': 'f3ec9bdce453816f94283a15a47cb952',
-                 'release': '2.fc7',
+                 'release': '2.fc17',
                  'size': 1993385,
                  'version': '1.0.2.2'},
                 ]
         if id == 16059:  # for updateinfo.xml tests
-            rpms[0]['nvr'] = rpms[1]['nvr'] = 'TurboGears-1.0.2.2-3.fc7'
-            rpms[0]['release'] = rpms[1]['release'] = '3.fc7'
+            rpms[0]['nvr'] = rpms[1]['nvr'] = 'TurboGears-1.0.2.2-3.fc17'
+            rpms[0]['release'] = rpms[1]['release'] = '3.fc17'
```

Comment:
It seems like these lines are missing test coverage, which would imply that they weren't necessarily changed for the tests. In either case, they lacked test coverage before anyway and this is just test code so no big deal.